libvast/native-plugins/segment_store.cpp

```diff
@@ -190,7 +191,7 @@ system::store_actor::behavior_type passive_local_store(
         if (!seg) {
           VAST_ERROR("{} couldn't create segment from chunk: {}", *self,
                      seg.error());
-          self->send_exit(self, caf::exit_reason::unhandled_exception);
```

Comment:
in CAF 18.6 this enum value is removed

Reply:
Note that this doesn't have to be a `caf::exit_reason`, you can actually provide any error code here. I think it'd make a lot of sense to just use `caf::make_error(vast::ec::something, fmt::format("{} failed to create segment from chunk: {}", *self, seg.error()))` here.

Reply:
@dominiklohmann 
Good tip, i will change it as proposed